lvk/vulkan/VulkanUtils.cpp

```diff
@@ -585,9 +585,17 @@ lvk::Result lvk::compileShader(VkDevice device,
       .language = GLSLANG_SOURCE_GLSL,
       .stage = getGLSLangShaderStage(stage),
       .client = GLSLANG_CLIENT_VULKAN,
+#ifndef __APPLE__
       .client_version = GLSLANG_TARGET_VULKAN_1_3,
+#else
+      .client_version = GLSLANG_TARGET_VULKAN_1_2,
+#endif
       .target_language = GLSLANG_TARGET_SPV,
+#ifndef __APPLE__
       .target_language_version = GLSLANG_TARGET_SPV_1_6,
+#else
+      .target_language_version = GLSLANG_TARGET_SPV_1_5,
```

Comment:
SPIR-V Cross should be aware of SPIR-V 1.6 https://github.com/KhronosGroup/SPIRV-Cross/blob/main/spirv_parser.cpp#L71

What prevents using it with the latest WIP MoltenVK?

Reply:
If I set up 1.6 version, I get:
```
ERROR:
Validation layer:
 Validation Error: VUID-VkShaderModuleCreateInfo-pCode-01379 
 Object 0: handle = 0x10203ac18, type = 
 MessageID = 0x0 
 SPIR-V module not valid: Invalid SPIR-V binary version 1.6 for target environment SPIR-V 1.5 (under Vulkan 1.2 semantics). The Vulkan spec states: If pCode is a pointer to GLSL code, it must be valid GLSL code written to the GL_KHR_vulkan_glsl GLSL extension specification (https://vulkan.lunarg.com/doc/view/1.3.250.1/mac/1.3-extensions/vkspec.html#VUID-VkShaderModuleCreateInfo-pCode-01379) 
```